docs/technical_reference/tools/parameter_sweep.rst

```diff
@@ -12,15 +12,28 @@ Overview
 The parameter sweep tool systematically fixes variables or modifies
 mutable parameters on a flowsheet (or Pyomo ConcreteModel), optimizes
 the flowsheet, and reports user-specified results.
-
-For each item the user wants to change, they specify a "short" name
-as a key, and then a tuple specifying the Pyomo object to be changed,
-a lower limit, an upper limit, and the number of elements to be sampled 
-for this parameter between the lower limit and upper limit.
+The parameter sweep tool can be operated in one of two ways: sweeping a 
+fixed set of paramters, or allowing for random samples from a distribution.
+While different fixed or random sampling types can be combined, in a single
+parameter sweep the user must use either all fixed or all random sampling types.
```

Comment:
```suggestion
parameter sweep the user must use either all fixed or all random sampling types
```

Reply:
Why remove the period here? The parenthetical after this is its own sentence.

Reply:
No need to entertain this one. =)